.github/workflows/branch-build.yml

```
@@ -80,7 +81,7 @@ jobs:
 
       - name: Install BotKube
         env:
-          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN_V2 }}
```

```suggestion
          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN_V2 }}
```
can we change that into `secrets.SOCKET_SLACK_BOT_TOKEN` or `secrets.WS_SLACK_BOT_TOKEN`? IMO in this way it will be more descriptive what it really is.

---

Sure, I will do that